src/neuroglancer/datasource/boss/backend.ts

```diff
@@ -22,20 +22,22 @@ import {VolumeChunkSourceParameters, MeshSourceParameters} from 'neuroglancer/da
 import {VolumeChunkSource, VolumeChunk} from 'neuroglancer/sliceview/volume/backend';
 import {decodeJsonManifestChunk, decodeTriangleVertexPositionsAndIndices, FragmentChunk, ManifestChunk, MeshSource} from 'neuroglancer/mesh/backend';
 import {ChunkDecoder} from 'neuroglancer/sliceview/backend_chunk_decoders';
-import {decodeJpegChunk} from 'neuroglancer/sliceview/backend_chunk_decoders/jpeg';
+// import {decodeJpegChunk} from 'neuroglancer/sliceview/backend_chunk_decoders/jpeg';
 import {decodeBossNpzChunk} from 'neuroglancer/sliceview/backend_chunk_decoders/bossNpz';
 import {openShardedHttpRequest, sendHttpRequest} from 'neuroglancer/util/http_request';
 import {CancellationToken} from 'neuroglancer/util/cancellation';
 import {Endianness} from 'neuroglancer/util/endian';
 import {registerSharedObject, SharedObject} from 'neuroglancer/worker_rpc';
 
+// TODO (rodrilm): Add logic to use the jpeg header and decoder if the data is uint8 and use npygz if it is uint16
+// As of right now we will only use the npygz content type to ensure all types are supported.
 let chunkDecoders = new Map<string, ChunkDecoder>();
 chunkDecoders.set('npz', decodeBossNpzChunk);
-chunkDecoders.set('jpeg', decodeJpegChunk);
+// chunkDecoders.set('jpeg', decodeJpegChunk);
 
 let acceptHeaders = new Map<string, string>();
 acceptHeaders.set('npz', 'application/npygz');
-acceptHeaders.set('jpeg', 'image/jpeg');
+// acceptHeaders.set('jpeg', 'image/jpeg');
```

Comment:
Commenting this out alone didn’t work?

Reply:
@movestill For some reason even after commenting it out, something seemed to allow it to use the image/jpeg header to be accepted. I'll take a closer look

Reply:
I do believe this is the way to go in the long term anyways. I think it will be safer to use the npygz option over jpeg